linalg/densemat.cpp

```diff
@@ -1066,31 +953,19 @@ void dsyevr_Eigensystem(DenseMatrix &a, Vector &ev, DenseMatrix *evect)
       A[i] = data[i];
    }
 
-#ifdef MFEM_USE_SINGLE
-   ssyevr_( &JOBZ, &RANGE, &UPLO, &N, A, &LDA, &VL, &VU, &IL, &IU,
-#elif defined MFEM_USE_DOUBLE
-   dsyevr_( &JOBZ, &RANGE, &UPLO, &N, A, &LDA, &VL, &VU, &IL, &IU,
-#else
-   MFEM_ABORT("Floating point type undefined");
-#endif
-            &ABSTOL, &M, W, Z, &LDZ, ISUPPZ, &QWORK, &LWORK,
-            &QIWORK, &LIWORK, &INFO );
+   MFEM_LAPACK_PREFIX(syevr_)(&JOBZ, &RANGE, &UPLO, &N, A, &LDA, &VL, &VU, &IL,
+                              &IU, &ABSTOL, &M, W, Z, &LDZ, ISUPPZ, &QWORK,
+                              &LWORK, &QIWORK, &LIWORK, &INFO );
```

Comment:
Extra space after `INFO` can be removed, here and below.

Reply:
Fixed in e4b8584a16616f2b50422b85c5ba0fa30cd68c46